azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/ManagementClientAsync.java

```diff
@@ -972,7 +982,7 @@ public ManagementClientAsync(URI namespaceEndpointURI, ClientSettings clientSett
 
     /**
      * Disposes and closes the managementClient.
-     * @throws IOException
+     * @throws IOException if an I/O error occurs
```

Comment:
>I/O error occurs [](start = 33, length = 16)

More specific please.

Reply:
I got that description from the AsyncHttpClient dependency. 

---
In reply to: [260570847](https://github.com/Azure/azure-service-bus-java/pull/341#discussion_r260570847) [](ancestors = 260570847)